### PR TITLE
Add last updated display, `params.showLastUpdated` and update README.md

### PR DESCRIPTION
**Pull request by horochx**
Get the last updated via `.Lastmod` and use the `params.showLastUpdated` field to control display and hide. The default value is hidden and does not cause damaging changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,9 @@ paginate = 5
   centerTheme = false
   # set a custom favicon (default is a `themeColor` square)
   # favicon = "favicon.ico"
+  # set post to show the last updated
+  # If you use git, you can set `enableGitInfo` to `true` and then post will automatically get the last updated
+  showLastUpdated = true
```
Comment (panr) [inline README.md]: If we want to have this `false` by default, you should change the value to `false` or comment this line though.

File: layouts/_default/single.html [hunk 2]
```diff
@@ -30,6 +30,11 @@ <h1 class="post-title">
   <div class="post-content">
     {{ .Content }}
   </div>
+  {{ if $.Site.Params.showLastUpdated }}
```
Comment (panr) [inline layouts/_default/single.html]: Are you sure you want this at the end of a post? We could align this with a creation date.

Just asking ;-)

File: source/css/post.css [hunk 3]
```diff
@@ -22,6 +22,26 @@
     color: color-mod(var(--accent) a(70%));
   }
 
+  &-lastmod {
+    --border: 3px dotted var(--accent);
```
Comment (panr) [inline source/css/post.css]: I'm not sure about the heavy styling for this kind of element. Could you show me a screenshot of this?

Comment (horochx) [reply source/css/post.css]: This is the rendering of the desktop:
desktop

And the mobile:
mobile

We will find that if the update time and the creation time are on the same line, a line break will occur when the website is accessed via mobile phone. I don't know how to improve the design, so I simply put the update time at the end of the post. If there is a more appropriate treatment, please let me know, thank you : D

File: README.md (hunk 1 above)
Comment (horochx) [reply README.md]: I will change the value to `false` :-D

File: source/css/post.css (hunk 3 above)
Comment (panr) [reply source/css/post.css]: Very sorry for a late response! I totally forgot about some of the PRs 😞 

About the styling. What do you think about doing something like this instead:
`2020-02-21 [Modified: 2020-03-03] :: Radek`

We can use config file to set any string we want to replace `Modified` or `Updated` (it's up to you what it should be by the default).

I'd like to keep this as minimal as possible 🙏

**Comment by panr** [comment]
https://github.com/panr/hugo-theme-terminal/pull/88/commits/800a9e8316fd07a69107a5be9d90568244747692 – Could you revert this commit? This isn't necessary. We want to keep development files in the repo ;-)

File: source/css/post.css (hunk 3 above)
Comment (horochx) [reply source/css/post.css]: Never mind, I actually forgot about this PR 😉。I think your idea is much better than mine, I will make some changes later and resubmit.

Comment (horochx) [reply source/css/post.css]: I made some changes and now it looks like this:
demo

And I added a nullable option named `updatedDatePrefix` (the default is Updated).Do you think it needs to be modified? :D

**Comment by panr** [comment]
I'm closing this PR but I'm gonna merge your changes in the different branch. Thanks and sorry it took me so long ;-(

**Pull request closed without merging**